resources/views/posts/index.blade.php

```diff
@@ -0,0 +1,44 @@
+@extends('base')
+
+@section('main')
+<div class="row">
+   <div class="col-sm-12">
+     <h1 class="display-3">Posts</h1>
+     <button type="button" class="btn btn-success" style="margin-bottom:10px; margin-top:10px;">Add new Post</button>
```

Comment:
```suggestion
     <button type="button" class="btn btn-success mt-1 mb-1">{{ trans('post.add') }}</button>
```
Chuyển hết các style vào trong file css rồi include vào thay vì viết trực tiếp nhé a.
Với lại a thêm localization cho các text nữa nhé.

Reply:
done,localization a sẽ làm 1 pull sau nhé Mạnh

Reply:
resolved